File: problems/3038-maximum-number-of-operations-with-the-same-score-i/solution.java
class Solution {
    public int maxOperations(int[] nums) {
        int operations = 0;
        if (nums.length < 2) {
            return 0;
        }
        int targetSum = nums[0] + nums[1];
        operations++;
        for (int i = 2; i < nums.length - 1; i += 2) {
            if (nums[i] + nums[i + 1] == targetSum) {
                operations++;
            } else {
                break;
            }
        }
        return operations;
    }
}

File: problems/3038-maximum-number-of-operations-with-the-same-score-i/solution.py
class Solution:
    def maxOperations(self, nums: List[int]) -> int:
        if len(nums) < 2:
            return 0

        score = nums[0] + nums[1]
        count = 1
        i = 2

        while i < len(nums) - 1:
            if nums[i] + nums[i+1] == score:
                count += 1
                i += 2
            else:
                break
        return count